### PR TITLE
fix: handled the validateDOMNesting error

### LINKED ISSUE #1065: validateDOMNesting error within `BottomCarousel component`
Within the `BottomCarousel` component on the policy calculator page, I receive the following console error:

```
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
p
div
p
div
BottomCarousel@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:704:7
div
div
./src/layout/ResultsPanel.jsx/_c<@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:3345:73
LowLevelDisplay@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:8917:7
DisplayImpact@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:8821:7
FetchAndDisplayImpact@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:9246:108
PolicyOutput@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:11116:7
div
div
ThreeColumnPage@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:3727:7
PolicyPage@http://localhost:3000/static/js/src_layout_MobileCalculatorPage_jsx-src_pages_PolicyPage_jsx.chunk.js:5086:7
Suspense
RenderedRoute@http://localhost:3000/static/js/bundle.js:344157:7
Routes@http://localhost:3000/static/js/bundle.js:344777:7
PolicyEngine@http://localhost:3000/static/js/bundle.js:11828:22
Router@http://localhost:3000/static/js/bundle.js:344719:7
BrowserRouter@http://localhost:3000/static/js/bundle.js:342768:7 policy:13024:25
```

### PR DESCRIPTION
**Pull request by dotslashbit**
## Description

Please provide a summary of the changes and which issue is fixed. Please include relevant motivation and context.

closes #1065 

## Changes
There were nesting p elements, I have modified the structure of the elements to get the same output but now the outer element is a div

### Before
Screenshot 2024-01-01 at 6 53 26 AM

### After
Screenshot 2024-01-01 at 6 54 10 AM

## Tests

Please describe how the change has been tested.

### CODE REVIEW

**Comment by anth-volk** [comment]
This would fix the issue; however, at the moment, BottomCarousel is very much engineered to accept only text, a clear limitation of the component. Could you change the `bottomText` prop name to `bottomElements` or something similar and ensure that the app utilizes that prop properly (I think it's only invoked in two or three spots at the moment)? Otherwise I think it's unclear that that prop is being displayed as a div that other tags can sit within.

**Review by anth-volk — CHANGES_REQUESTED** [review]
Please see commentary above

**Comment by dotslashbit** [comment]
@anth-volk I have updated the code, let me know if there's any correction to be made

File: src/pages/household/output/HouseholdOutput.jsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ export default function HouseholdOutput(props) {
         <BottomCarousel
           selected={focus}
           options={HOUSEHOLD_OUTPUT_TREE[0].children}
-          bottomText="PolicyEngine results may not constitute exact tax liabilities or benefit entitlements."
+          bottomElements="PolicyEngine results may not constitute exact tax liabilities or benefit entitlements."
```
Comment (anth-volk) [inline src/pages/household/output/HouseholdOutput.jsx:152]:
```suggestion
          bottomElements={<p>PolicyEngine results may not constitute exact tax liabilities or benefit entitlements.</p>}
```

Comment (anth-volk) [reply src/pages/household/output/HouseholdOutput.jsx:152]: Apologies, should've also mentioned this before in the earlier review.

**Review by anth-volk — CHANGES_REQUESTED** [review]
Thanks again, @dotslashbit. I left one comment that I should've flagged last time and neglected that is meant to keep the functioning of this prop standardized. After that, I think it's good to merge.

**Comment by dotslashbit** [comment]
@anth-volk did the required changes, let me know if there's anything needed to be done

**Review by anth-volk — APPROVED** [review]
This is perfect, thanks so much @dotslashbit!

**Pull request merged**